pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Kafka bindings
+description: Learn how to configure Kafka bindings in your AsyncAPI document.
+weight: 300
+---
+
+## Introduction
+
+In a previous tutorial, you learned how to manage schemas with a schema registry. This tutorial teaches you how Kafka bindings function by defining Kafka messages and expanding your AsyncAPI document with protocol-specific details.
```

Comment:
@CynthiaPeter could you link the previous tutorial here?

Reply:
Thank you, Vaishnavi. Done.